storey/flow.py

```diff
@@ -1566,6 +1582,8 @@ def __init__(
 
         self._process_executor_by_runnable_name = {}
 
+        self.monitored = kwargs.get("track", False)
```

Comment:
I think you can add it to the parameter list explicitly. And maybe decide on one name vis-a-vis `monitored` vs `tracked`?